chapter2/tests/t5/chapter2-l2-t5-get-file-unique-extensions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# The script gets a directory as an argument and returns a list of unique extensions of all files (including in the nested directories).
+set -eou pipefail
+
+IFS=$'\n'
+
+#######################################
+# Get directory files extensions (including nested directories).
+# Arguments:
+#   Directory.
+# Outputs:
+#   Writes extension into stdout.
+#######################################
+function get_files_extensions {
+  for file in $(find "$1" -type f 2>/dev/null); do
```

Comment:
Let's add a separate local named variable for `$1` above loop for better readability